packages/admin/admin/src/actionLog/diffHeader/DiffHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import { type Box, type ComponentsOverrides, type Theme, type Typography } from "@mui/material";
+import { useThemeProps } from "@mui/material/styles";
+import { FormattedDate, FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { DateTypography, InfoTypography, Root, UserNameTypography, VersionTypography } from "./DiffHeader.sc";
+
+export type DiffHeaderClassKey = "root" | "versionTypography" | "userNameTypography" | "dateTypography" | "infoTypography";
```

Comment:
I don't think the underlying component should necessarily be mentioned in the slots name. 

```suggestion
export type DiffHeaderClassKey = "root" | "version" | "userName" | "date" | "info";
```